pkg/client/didexchange/client.go

```diff
@@ -113,7 +113,7 @@ func New(ctx provider) (*Client, error) {
 // CreateInvitation creates an invitation. New key pair will be generated and base58 encoded public key will be
 // used as basis for invitation. This invitation will be stored so client can cross reference this invitation during
 // did exchange protocol.
-func (c *Client) CreateInvitation(label string) (*Invitation, error) {
+func (c *Client) CreateInvitation(label, routerConnID string) (*Invitation, error) {
```

Comment:
It would be better to pass it as an option or create new API `CreateInvitation with router`. Having second argument makes passing routerConnID mandatory (either empty or some value) for the consumer, which ideally be shouldn't because framework is deriving the default router.

Reply:
Agree. I did it to understand who is using the function. I will make it optional.

Reply:
Done!